internal/adapter/lsp/document.go

```diff
@@ -7,12 +7,12 @@ import (
 	"strings"
 	"unicode/utf16"
 
+	"github.com/tliron/glsp"
+	protocol "github.com/tliron/glsp/protocol_3_16"
 	"github.com/zk-org/zk/internal/core"
 	"github.com/zk-org/zk/internal/util"
 	"github.com/zk-org/zk/internal/util/errors"
 	strutil "github.com/zk-org/zk/internal/util/strings"
-	"github.com/tliron/glsp"
-	protocol "github.com/tliron/glsp/protocol_3_16"
```

Comment:
why the reordering? Did gofmt do this?

Reply:
yes. Same as the spaces / tabs you pointed out above... I went back to double check and ran `go fmt` again.

Reply:
sounds good. I'm fine with unrelated changes like this if it wasn't done properly before (i.e. `go fmt` wasn't run), but if its just a matter of preference, I'd rather keep the git blame clean. Looks like we're good to keep this

Reply:
Nice, me too. I appreciate that level of attention / scrutiny 🔥